test/http-data-source.test.ts

```diff
@@ -1336,6 +1336,56 @@ test('Should only cache GET successful responses with the correct status code',
   t.is(cacheMap.size, 0)
 })
 
+test('Response LRU cache will clear after 100ms when user set maxAge to 100ms', async (t) => {
```

Comment:
I'd rename it to `Global maxAge should be used when no maxAge was set` or similar.

Reply:
agree